compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -9672,25 +9675,50 @@ OMR::Z::TreeEvaluator::loadaddrEvaluator(TR::Node * node, TR::CodeGenerator * cg
                cursor = generateRILInstruction(cg, TR::InstOpCode::LARL, node, targetRegister, node->getSymbolReference(),
                                                 reinterpret_cast<void*>(sym->getStaticAddress()),  NULL);
                }
+            else if (cg->needRelocationsForBodyInfoData && sym && sym->isRecompilationCounter())
```

Comment:
Missing parenthesis `cg->needRelocationsForBodyInfoData`

Reply:
Sorry missed that one! Fixed here: c876084fb7c108efbd6fe296bd1ba0faee7ad22e